test/lib/fasta/indexedFasta.js

```diff
@@ -104,7 +104,7 @@ export default class IndexedFasta {
     }
 
     let residues = Buffer.allocUnsafe(readlen)
-    await this.data.read(residues, 0, readlen, position)
+    await this.data.read(headbytes, 0, cramFileDefinitionParser.maxLength, 0)
```

Comment:
what is this? doesn't seem correct

Reply:
wow, I think at some point webstorm went too intelligent on me and screwed up a bunch of .read() function calls around the project. what is a bit strange - this code is in the test folder, but it does not seem to have any impact on tests outcome...